### PR TITLE
Adds unit tests for CounterBadge

### PR DESCRIPTION
**Pull request by procload**
## New Behavior

Adds tests for CounterBadge

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by halerankin — APPROVED** [review]

**Review by brianchristopherbrady — APPROVED** [review]

**Review by miroslavstastny — COMMENTED** [review]
Multiple flaky tests are reported:
```
8 flaky
    [chromium] › src/counter-badge/counter-badge.spec.ts:73:3 › CounterBadge component › should show 0 when showZero is set programmatically 
    [chromium] › src/counter-badge/counter-badge.spec.ts:89:3 › CounterBadge component › should not show 0 when showZero is set programmatically 
    [chromium] › src/counter-badge/counter-badge.spec.ts:121:3 › CounterBadge component › should show and hide dot programmatically 
    [firefox] › src/counter-badge/counter-badge.spec.ts:73:3 › CounterBadge component › should show 0 when showZero is set programmatically 
    [firefox] › src/counter-badge/counter-badge.spec.ts:89:3 › CounterBadge component › should not show 0 when showZero is set programmatically 
    [firefox] › src/counter-badge/counter-badge.spec.ts:121:3 › CounterBadge component › should show and hide dot programmatically 
    [webkit] › src/counter-badge/counter-badge.spec.ts:73:3 › CounterBadge component › should show 0 when showZero is set programmatically 
    [webkit] › src/counter-badge/counter-badge.spec.ts:89:3 › CounterBadge component › should not show 0 when showZero is set programmatically 

```

File: packages/web-components/src/counter-badge/counter-badge.spec.ts [hunk 1]
```diff
@@ -96,6 +94,7 @@ test.describe('CounterBadge component', () => {
 
     const shadowContent = await page.evaluate(() => {
       const element = document.querySelector('fluent-counter-badge');
+      console.log(element);
```
Comment (miroslavstastny) [inline packages/web-components/src/counter-badge/counter-badge.spec.ts:97]: remove the `console.log()`

**Comment by miroslavstastny** [comment]
I still see flaky tests:
```
  2 flaky
    [chromium] › src/counter-badge/counter-badge.spec.ts:120:3 › CounterBadge component › should show and hide dot programmatically 
    [firefox] › src/counter-badge/counter-badge.spec.ts:120:3 › CounterBadge component › should show and hide dot programmatically 
```

**Comment by miroslavstastny** [comment]
Still red:
```
  2 failed
    [firefox] › src/counter-badge/counter-badge.spec.ts:115:3 › CounterBadge component › should hide dot programmatically 
    [webkit] › src/counter-badge/counter-badge.spec.ts:115:3 › CounterBadge component › should hide dot programmatically 
  2 flaky
    [chromium] › src/counter-badge/counter-badge.spec.ts:71:3 › CounterBadge component › should show 0 when showZero is set programmatically 
    [firefox] › src/counter-badge/counter-badge.spec.ts:71:3 › CounterBadge component › should show 0 when showZero is set programmatically 
```

File: packages/web-components/.storybook/main.cjs [hunk 2]
```diff
@@ -26,7 +26,7 @@ module.exports = /** @type {Omit<import('../../../.storybook/main').StorybookCon
         backgrounds: false,
         viewport: false,
         toolbars: false,
-        actions: false,
```
Comment (miroslavstastny) [inline packages/web-components/.storybook/main.cjs:29]: Is this intentional or just a debugging left over?

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**